src/document/DocumentAnnotator.ts

```diff
@@ -97,10 +97,24 @@ export default class DocumentAnnotator extends BaseAnnotator {
             return;
         }
 
-        if (mode === Mode.HIGHLIGHT) {
-            this.annotatedEl.classList.add('ba-is-highlighting');
-        } else {
-            this.annotatedEl.classList.remove('ba-is-highlighting');
+        // Clear all existing classes from the element
```

Comment:
This is not safe, as the annotated element may have other classes that are not managed by this SDK.